### ISSUE TITLE
[TF-TD] Namespaces used in the TD / WoT ontology 

### ISSUE THREAD

**Issue by danicic** [w3c/wot-thing-description#3]
In response to discussion on namespaces used in the TD / WoT ontology, let us start with the following list of namespaces:
 
td <http://www.w3.org/ns/td#> 
rdfs      <http://www.w3.org/2000/01/rdf-schema#>
xsd      <http://www.w3.org/2001/XMLSchema#>
owl       <http://www.w3.org/2002/07/owl#>
rdf        <http://www.w3.org/1999/02/22-rdf-syntax-ns#>

Note that in the current version of the [WoT ontology](http://iot.linkeddata.es/def/wot/index-en.html) the **wot** prefix is used instead of **td**. The task force should agree on the name of the core vocabulary, i.e., **td** or **wot**.

Later we may add additional namespaces as we progress with the activity.

**Comment by danielpeintner** [comment]
We may also want to check the latest W3C policy w.r.t. how a namespace must look like

- starting with year?
- starting with year and month?
 
et cetera

**Comment by danicic** [comment]
@prefix td: <http://www.w3.org/ns/td#<http://www.w3.org/ns/td>> is conformant with the W3C policy, and we can use it as the TD namespace.

Cheers, Darko

Von: danielpeintner [mailto:notifications@github.com]
Gesendet: Freitag, 21. April 2017 08:56
An: w3c/wot-thing-description
Cc: Anicic, Darko (CT RDA NEC WOS-DE); Author
Betreff: Re: [w3c/wot-thing-description] [TF-TD] Namespaces used in the TD / WoT ontology (#3)


We may also want to check the latest W3C policy w.r.t. how a namespace must look like

  *   starting with year?
  *   starting with year and month?

et cetera

**Comment by sebastiankb** [comment]
still, there is the question about the different versions of TDs in the future. E.g., TD 1.0 version will be releases in 2018 and an updated TD 2.0 version in 2023. How we make the differentiation by using this generic namespace?

**Comment by mmccool** [comment]
I would think the form of the URL matters more than the "td" or "wot", since normally these are just shorthands in particular files.   It's nice if they are consistent for readability, but not crucial.

But the URL should definitely include the year, and maybe with a version number as well, and maybe with both WoT and td for good measure, eg something like 

http://www.w3.org/2018/01/WoT-td

**Comment by sebastiankb** [comment]
Based on the Osaka meeting we should postpone the discussion about the TD namespace until the TD specification is finally released. For now we stay with http://www.w3.org/ns/td 

I missing the continuation of the origin discussion about the TD vocabulary (name, inputData, mediaType, ...) and which common namespaces they should equivalent to. Is there any news in iot.schema.org about this?

**Comment by mkovatsc** [comment]
@vcharpenay Didn't you look into this as well? Last time I heard about this it was pending on @draggett IIRC.

**Comment by draggett** [comment]
Following precedent, the idea is to define our core vocabulary under https://www.w3.org/ns/td
We don't need Director approval for this as I believe it is within my remit as the lead for the W3C Data activity.

**Comment by sebastiankb** [comment]
I will close this non active issue and point to the new issue #149 that reflects the latest TD version.
